src/test/regress/expected/multi_insert_select.out

```diff
@@ -2392,6 +2392,47 @@ SELECT * FROM ref_table ORDER BY user_id, value_1;
 (5 rows)
 
 DROP TABLE ref_table;
+-- Select from reference table into reference table
+CREATE TABLE ref_table_source (user_id int, value_1 int);
+SELECT create_reference_table('ref_table_source');
+ create_reference_table 
+------------------------
+ 
+(1 row)
+
+CREATE TABLE ref_table_destination (user_id int, value_1 int);
+SELECT create_reference_table('ref_table_destination');
+ create_reference_table 
+------------------------
+ 
+(1 row)
+
+INSERT INTO ref_table_source
+SELECT user_id, value_1 FROM raw_events_second;
+SELECT * FROM ref_table_source ORDER BY user_id, value_1;
+ user_id | value_1 
+---------+---------
+       1 |        
+       2 |        
+       3 |        
+       4 |        
+       5 |        
+(5 rows)
+
+INSERT INTO ref_table_destination
+SELECT user_id, value_1 FROM ref_table_source;
```

Comment:
Fix looks good, but the tests don't exercise the logic. This issue only happens when go through  function evaluation (e.g. there is a `now()` in the query, or a cast that is implemented as a stable function - see #1688)

Reply:
oh right, I tempted to use existing tables in regressions. I updated the regressions.